modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx

```diff
@@ -48,7 +48,7 @@ export function ConfigurationContainer({
 					}
 				}}
 				onToggle={() => setValues({portlet: !values.portlet})}
-				toggled={values.portlet}
+				toggled={values.portlet && !isRootDescendantNode}
```

Comment:
Hey @larissacribeiro, thanks for this PR!

I believe you need to change the value of values.portlet in the data structure. And not just render the toggle as disabled.

![screenshot-1698700756](https://github.com/liferay-objects/liferay-portal/assets/61854510/d2fa9540-2f28-46c5-babb-f9d3f1dfab8e)

Reply:
The main objective is to understand why this value is true in the data structure. And improve the logic to set this to false when Biden is created.

CC: @larissacribeiro

Reply:
@igor-franca Just updated it! thanks 😄